suggestions.go

```diff
@@ -16,13 +16,89 @@ type SuggestFlagFunc func(flags []Flag, provided string, hideHelp bool) string
 
 type SuggestCommandFunc func(commands []*Command, provided string) string
 
+// jaroDistance is the measure of similarity between two strings. The result is
+// 1 for equal strings, and 0 for completely different strings.
```

Comment:
I'd add that the result is _between_ 0 or 1. This description at first made me think the output is either 0 or 1, only.